### PR TITLE
Merge upstream/momza 1063

### PR DESCRIPTION
**Pull request by DevChima**
Merge upstream changes to staging
Fixed errors

### CODE REVIEW

File: bin/python2.7 [hunk 1]
```diff
@@ -0,0 +1 @@
+python
```
Comment (erikh360) [inline bin/python2.7]: This bin directory can be removed

File: include/python2.7 [hunk 2]
```diff
@@ -0,0 +1 @@
+/usr/include/python2.7
```
Comment (erikh360) [inline include/python2.7]: This include dir can also be removed.

Comment (DevChima) [reply include/python2.7]: Done

File: bin/python2.7 (hunk 1 above)
Comment (DevChima) [reply bin/python2.7]: Removed

**Pull request merged**